src/createServer.js

```diff
@@ -1,3 +1,36 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+
+const { convertToCase } = require('./convertToCase');
+const { parseURL } = require('./parseURL');
+const { validateData } = require('./validateData');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const { stringCase, queryString } = parseURL(req.url, req.headers.host);
+    const validation = validateData(queryString, stringCase);
+
+    res.writeHead(200, { 'Content-Type': 'application/json' });
+
+    if (!validation.errors.length) {
+      const result = {
```

Comment:
would be better to change it
First, you check for errors
If you don't have an error - you make a succusses response